pyvisa/resources/serial.py

```diff
@@ -21,55 +21,62 @@ class SerialInstrument(MessageBasedResource):
     baud_rate: Attribute[int] = attributes.AttrVI_ATTR_ASRL_BAUD()
 
-    #: Number of data bits contained in each frame (from 5 to 8).
+    #: Number of data bits contained in each frame (from 5 to 8). The default value is 8.
     data_bits: Attribute[int] = attributes.AttrVI_ATTR_ASRL_DATA_BITS()
 
-    #: Parity used with every frame transmitted and received.
+    #: Parity used with every frame transmitted and received. The default value is None (VI_ASRL_PAR_NONE).
     parity: Attribute[constants.Parity] = attributes.AttrVI_ATTR_ASRL_PARITY()
 
-    #: Number of stop bits used to indicate the end of a frame.
+    #: Number of stop bits used to indicate the end of a frame. The default value is 1 (VI_ASRL_STOP_ONE).
     stop_bits: Attribute[constants.StopBits] = attributes.AttrVI_ATTR_ASRL_STOP_BITS()
 
-    #: Indicates the type of flow control used by the transfer mechanism.
+    #: Indicates the type of flow control used by the transfer mechanism. The default value
+    #: is None (VI_ASRL_FLOW_NONE).
     flow_control: Attribute[
         constants.ControlFlow
     ] = attributes.AttrVI_ATTR_ASRL_FLOW_CNTRL()
 
     #: Number of bytes available in the low- level I/O receive buffer.
     bytes_in_buffer: Attribute[int] = attributes.AttrVI_ATTR_ASRL_AVAIL_NUM()
 
-    #: If set to True, NUL characters are discarded.
+    #: If set to True, NUL characters are discarded. The default is False.
     discard_null: Attribute[bool] = attributes.AttrVI_ATTR_ASRL_DISCARD_NULL()
 
-    #: Manually control transmission.
+    #: Manually control transmission. The default value is True.
     allow_transmit: Attribute[bool] = attributes.AttrVI_ATTR_ASRL_ALLOW_TRANSMIT()
 
-    #: Method used to terminate read operations.
+    #: Method used to terminate read operations. The default method terminates by
+    #: searching for "/" appending the termination character (VI_ASRL_END_TERMCHAR).
     end_input: Attribute[
         constants.SerialTermination
     ] = attributes.AttrVI_ATTR_ASRL_END_IN()
 
-    #: Method used to terminate write operations.
+    #: Method used to terminate write operations. The default method terminates
+    #: when all requested data is transferred or when an error occurs (VI_ASRL_END_NONE).
     end_output: Attribute[
         constants.SerialTermination
     ] = attributes.AttrVI_ATTR_ASRL_END_OUT()
 
-    #: Duration (in milliseconds) of the break signal.
+    #: Duration (in milliseconds) of the break signal. The default value is 250.
     break_length: Attribute[int] = attributes.AttrVI_ATTR_ASRL_BREAK_LEN()
 
-    #: Manually control the assertion state of the break signal.
+    #: Manually control the assertion state of the break signal. The default state is
+    #: VI_STATE_UNASSERTED.
     break_state: Attribute[
         constants.LineState
     ] = attributes.AttrVI_ATTR_ASRL_BREAK_STATE()
 
     #: Character to be used to replace incoming characters that arrive with errors.
+    #: The default character is '\0'.
```

Comment:
I'm not sure if I interpreted this accurately, can you advise? @MatthieuDartiailh

Reply:
I believe it is right. It is similar to XON and XOFF